storage/innobase/include/api0api.h

```diff
@@ -651,22 +651,26 @@ ib_open_table_by_name(
 Insert a row to a table.
 @return	DB_SUCCESS or err code */
 
+#pragma namemanglingrule(fnparmtype, on)
```

Comment:
That will produce a warning ( actually, tons of warnings, since it is a popular header file) about unknown pragma on some compilers. It makes sense to surround it with ifdef _AIX